crates/rpc/rpc/src/eth/filter.rs

```diff
@@ -67,7 +67,7 @@ where
 
     async fn uninstall_filter(&self, id: SubscriptionId<'_>) -> RpcResult<bool> {
```

Comment:
nit: also FilterId

Reply:
On other methods too I guess? (`filter_logs()` and `filter_changes()`)

Reply:
yep let's use FilterId consistently, everywhere